backend/chainlit/types.py

```diff
@@ -33,8 +33,8 @@ class Pagination(BaseModel):
 
 
 class ThreadFilter(BaseModel):
-    feedback: Optional[Literal[-1, 0, 1]] = None
-    userIdentifier: Optional[str] = None
+    feedback: Optional[Literal[0, 1]] = None
+    userId: Optional[str] = None
```

Comment:
@willydouhard why is `-1` being removed?

Reply:
-1 becomes 0 and we introduce the concept of "feedback deletion" to replace the current 0 purpose. Having 3 values was a workaround to avoid to handle feedback deletion